test/IRGen/section.swift

```diff
@@ -38,6 +38,8 @@
 // ASM-NOT: .section
 // ASM: $s7section3fooyyF:
 // ASM-linux-gnu: .section{{.*}}__TEXT,__mysection
+// ASM-linux-android: .section{{.*}}__TEXT,__mysection
+// ASM-linux-androideabi: .section{{.*}}__TEXT,__mysection
```

Comment:
This looks great.

Reply:
I have now tested this and made sure it works.